### PR TITLE
Normalize all resource names in tf and tfstate files

### LINKED ISSUE #277: aws_iam_user_policy resource name can only contain letters, numbers, dashes, and underscores.
I used terraforming to extract the configuration of my account in AWS (which had been built artisanally), and later **terraform plan** complained that one users policy which had his user name as "user.name" had the above listed problem.

Checking in the console indicated that the Policy Name actually had a period in it, so I'm not sure if the problem is in Terraform or Terraforming.

### PR DESCRIPTION
**Pull request by nabarunchatterjee**
Fixes #277

### CODE REVIEW

File: lib/terraforming/resource/iam_policy_attachment.rb [hunk 1]
```diff
@@ -77,6 +77,11 @@ def iam_policy_attachments
           }
         end
       end
+
+      def module_name_of(policy_attachment)
+        normalize_module_name(policy_attachment[:name])
+      end
+
```
Comment (dtan4) [inline lib/terraforming/resource/iam_policy_attachment.rb]: Please remove this empty line.

**Comment by nabarunchatterjee** [comment]
Could you please review and merge this. This is critical for the work I am doing. Thanks.

**Comment by dtan4** [comment]
LGTM 👍 Sorry for late!

**Comment by dtan4** [comment]
Released as v0.13.0.

**Pull request merged**